firefox-ios/firefox-ios-tests/Tests/XCUITests/InactiveTabsTest.swift

```diff
@@ -0,0 +1,137 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+final class InactiveTabsTest: BaseTestCase {
+    override func setUp() {
+        launchArguments = [
```

Comment:
I think there is something missing here... before this you would need to specify which is the test that is going to load the database, see for example:
https://github.com/mozilla-mobile/firefox-ios/blob/4d443cd21429767aaad91c5b3b81d6e5d5bdfc7f/firefox-ios/firefox-ios-tests/Tests/XCUITests/ActivityStreamTest.swift#L22

Reply:
There is only one test in this file so far. In addition, the database does not contain tabs. A database contains history and bookmarks.

Tabs archive is in a separate file.